src/profile/scripts/routes.js

```diff
@@ -12,9 +12,9 @@
                     controller: 'ProfileCtrl',
                     anonymous: false
                 })
-                .when('/delete-profile-confirmation/', {
-                    templateUrl: 'templates/user_management/profile/delete-profile-confirmation.html',
-                    controller: 'ProfileDeletedCtrl'
+                .when('/delete-account-confirmation/', {
```

Comment:
These changes are confusing because all files still live in the `profile` folder and I'm not sure if the new name is helpful?